tests/clients/test_fenda_client.py

```diff
@@ -85,12 +85,15 @@ def test_computing_loss(get_fenda_client: FendaClient) -> None:  # noqa
         "old_global_features": old_global_features,
         "aggregated_global_features": aggregated_global_features,
     }
+
     training_loss = fenda_client.compute_training_loss(preds=preds, target=target, features=features)
     evaluation_loss = fenda_client.compute_evaluation_loss(preds=preds, target=target, features=features)
     assert isinstance(training_loss.backward["backward"], torch.Tensor)
     assert pytest.approx(0.8132616, abs=0.0001) == evaluation_loss.checkpoint.item()
-    assert evaluation_loss.checkpoint.item() != training_loss.backward["backward"].item()
```

Comment:
Do you mind if we reinstate a check like this? It could simply be 
```
assert training_loss.additional_losses["loss"].item() == training_loss.backward["backward"].item()
```
We previously had a bit of a nasty bug where the two tensors shared memory and inadvertently ended up being the same value. This ensures that isn't happening.